src/applications/vaos/appointment-list/components/ReviewRequestsAndReferrals.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import classNames from 'classnames';
+
+export default function ReviewRequestsAndReferrals() {
```

Comment:
I would name this a little more specifically. Something like `RequestsAndReferralsLink`. I feel like that would make it quick to identify in the list.

Reply:
Moving this from a component to the mark up being in the page so no longer relevant, but I agree with comments.